html-relative-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Contact Us</title>
+  <style>
+    img {
+      width: 300px;
+    }
+
+  </style>
+</head>
+
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
+    <li><a href="../projects/projects.html">Projects</a></li>
+    <li><a href="../contact-us/contact-us.html">Contact Us</a></li>
```

Comment:
Whoops! Looks like you are exiting the folder to just go right back into it! In this case, you want the `contact-us.html` file, which is the same file you are in, which means it is in the same directory, so you can just say the name of the file directly.

Reply:
Oh, I just forgot to change it after I copied it from the other html index file of mine. I will fix and go grab some coffee to wake up, haha